function accept (req) {
          throw new Error('oops!')
        }